template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Extensions/AzureAd/AzureAdServiceCollectionExtensions.cs

```diff
@@ -40,7 +40,7 @@ public PostConfigureAzureOptions(IOptions<AzureAdOptions> azureOptions)
 
             public void PostConfigure(string name, JwtBearerOptions options)
             {
-                options.Audience = _azureOptions.ClientId;
+                options.Audience = _azureOptions.Audience;
```

Comment:
@HaoK Can you confirm that our change here to use Audience is the right thing to still do for AD Org Auth, this would make it consistent with 1.1 Org Auth. 

For Individual Auth B2C we are passing the ClientId to options.Audience.